src/enums.js

```diff
@@ -90,13 +91,19 @@ export const itemType = enumify([
   'SPRINKLER',
   'STONE',
   'TOOL_UPGRADE',
+  'WEEDS',
 ])
 
 /**
  * @property farmhand.module:enums.fertilizerType
  * @enum {string}
  */
-export const fertilizerType = enumify(['NONE', 'STANDARD', 'RAINBOW'])
+export const fertilizerType = enumify([
+  'NONE',
+  'STANDARD',
+  'RAINBOW',
+  'COMPOST',
```

Comment:
If we drop the mechanic of using compost as fertilizer directly, I think we can drop this addition.